### PR TITLE
downgrade pyside from 6.4.1 to 6.3.2 [CPP-905]

### PR DESCRIPTION
**Pull request by adrian-kong**
settings table scrolling is laggy on 6.4.1, downgrading back to 6.3.2 works fine.

should have no compatibility issues reverting.

### CODE REVIEW

**Comment by adrian-kong** [comment]
ignore failing CI - codesigning QtWebEngine

File: Makefile.toml [hunk 1]
```diff
@@ -90,7 +90,7 @@ PY_BASE_URL = "https://github.com/indygreg/python-build-standalone/releases/down
 SWFT_PY_BASE_URL = "https://github.com/swift-nav/python-build-standalone/releases/download/20220205%2Bswift"
 MACOSX_DEPLOYMENT_TARGET = "10.15"
 APP_NAME = "swift-console"
-QT_VERSION = "6.4.1"                                                                                         # Also needs to be updated in .github/workflows/main.yml
+QT_VERSION = "6.3.2"                                                                                         # Also needs to be updated in .github/workflows/main.yml
```
Comment (silverjam) [inline Makefile.toml:93]: Is the comment here still true?  If so, we should update .github/workflows/main.yaml too.

Comment (silverjam) [reply Makefile.toml:93]: Otherwise, let's remove the comment.

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4258017079)

File: Makefile.toml (hunk 1 above)
Comment (adrian-kong) [reply Makefile.toml:93]: ah forgot about the workflow as well.
updated

**Review by silverjam — APPROVED** [review]

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/4258416059)

**Pull request merged**